src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializerBase.java

```diff
@@ -411,7 +432,21 @@ protected BeanDeserializerBase(BeanDeserializerBase src, BeanPropertyMap beanPro
 
     public abstract BeanDeserializerBase withObjectIdReader(ObjectIdReader oir);
 
-    public abstract BeanDeserializerBase withIgnorableProperties(Set<String> ignorableProps);
+    public BeanDeserializerBase withIgnorableProperties(Set<String> ignorableProps) {
+        return withIgnorableProperties(ignorableProps, _includableProps);
+    }
+
+    /**
+     * @since 2.12
+     */
+    public abstract BeanDeserializerBase withIgnorableProperties(Set<String> ignorableProps, Set<String> includableProps);
```

Comment:
(note to self, mostly)

Makes sense to use different name, I think, `withPropertyInclusions()` (to indicate both ignoral and/or inclusion  contained), for the new method.